prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
```

Comment:
@trentm np.  I'm curious as to why there is a preference.  I haven't read anything to suggest it matters to asciidoc.